app/src/main/java/com/futurice/freesound/viewmodel/DisplayableItem.java

```diff
@@ -0,0 +1,46 @@
+package com.futurice.freesound.viewmodel;
```

Comment:
Hmm, we can see how it works out. I'm thinking of modifying the package structure a bit to make it more suitable for modules in any case.